entity-types/ext-kiosk/definition.yml

```diff
@@ -0,0 +1,43 @@
+domain: EXT
+type: KIOSK
+synthesis:
+  name: device.number
+  identifier: device.number
+  conditions:
+    - attribute: eventType
+      value: Logs
+    - attribute: newrelic.source
+      value: "api.logs"
+    - attribute: store.number
+      present: true
+    - attribute: device.type
+      value: CSO # CSO === KIOSK
+  tags:
+    State:
+      multiValue: false
+    Version:
+      multiValue: false
+    Release:
+      multiValue: false
+    country:
+      multiValue: false
+    device.type:
+      multiValue: false
+    geo.location.lat:
+      entityTagName: lat
+      multiValue: false
+    geo.location.lon:
+      entityTagName: lon
+      multiValue: false
+    host.ip:
+      entityTagName: ip
+      multiValue: false
+    host.name:
+      multiValue: false
+# dashboardTemplates:
+#   newRelic:
+#     template: dashboard.json
+configuration:
+  # enable only when things are confirmed so we can at least release the old entities whilst testing
+  # entityExpirationTime: EIGHT_DAYS
```

Comment:
TTL is required when using synthesis so even if not provided it will default to 8 days.